README.md

```diff
@@ -14,6 +14,22 @@ The template supports producing both Hungarian and English theses, which can be
 
 ## Compilation
 
+Compilation might be carried out through a preferred IDE (e.g. [TexStudio](https://www.texstudio.org/)), given the same commands should be executed.
```

Comment:
It is good that is remark was moved here, but *"given the same commands should be executed"* is now ambiguous to what is refers. Maybe it could be stated that the commands in the *Manually* section. (I don't know whether TexStudio supports latexmk.)